pkg/model/image_manager_test.go

```diff
@@ -31,3 +31,33 @@ func TestImageManager_test_defining_image_using_environment_variable(t *testing.
 	//then
 	assert.Equal(t, "test", imageChooser.Images[KeycloakImage])
 }
+
+func TestImageManager_test_overriding_rhsso_image_using_environment_variable(t *testing.T) {
+	//given
+	os.Setenv(RHSSOImage, "test")
+
+	//when
+	imageChooser := NewImageManager()
+	os.Unsetenv(RHSSOImage)
+
+	//then
+	assert.Equal(t, "test", imageChooser.Images[RHSSOImage])
+}
+
+func TestImageManager_test_overriding_multiple_images_using_environment_variables(t *testing.T) {
+	//given
+	os.Setenv(RHSSOImage, "RHSSOImage")
+	os.Setenv(RHSSOImageOpenJ9, "RHSSOImageOpenJ9")
+	os.Setenv(RHSSOImageOpenJDK, "RHSSOImageOpenJDK")
+
+	//when
+	imageChooser := NewImageManager()
+	os.Unsetenv(RHSSOImage)
+	os.Unsetenv(RHSSOImageOpenJ9)
+	os.Unsetenv(RHSSOImageOpenJDK)
+
+	//then
+	assert.Equal(t, "RHSSOImage", imageChooser.Images[RHSSOImage])
+	assert.Equal(t, "RHSSOImageOpenJ9", imageChooser.Images[RHSSOImageOpenJ9])
+	assert.Equal(t, "RHSSOImageOpenJDK", imageChooser.Images[RHSSOImageOpenJDK])
```

Comment:
@slaskawi Could you please add the following line to this test:
```go
assert.Equal(t, "RHSSOImage", imageChooser.getRHSSOImage())
```

Reply:
Added. Although, I'm not sure if it makes sense... 

The `getRHSSOImage` method has been written with lower case, which means it's not-exported (an equivalent to Java's private/protected in some sense).

Reply:
I see, you are right, I missed that the function is used to obtain `imageChooser.Images[RHSSOImage]`. Sorry about that